src/testUtils.ts

```diff
@@ -16,7 +16,7 @@ const sendSignal = 'SIGKILL';
 const outputChannel = vscode.window.createOutputChannel('Go Tests');
 const statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
 statusBarItem.command = 'go.test.cancel';
-statusBarItem.text = 'Cancel Running Tests';
+statusBarItem.text = '$(x) Cancel Running Tests';
```

Comment:
How does `${x}` relate the icon and how did you find it :) ?

Reply:
Here https://code.visualstudio.com/docs/extensions/example-word-count#_customizing-the-status-bar

We can use https://octicons.github.com/ on vscode

Reply:
Nice!